parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetOutputFormat.java

```diff
@@ -108,7 +111,11 @@
   public static final String WRITER_VERSION       = "parquet.writer.version";
   public static final String ENABLE_JOB_SUMMARY   = "parquet.enable.summary-metadata";
   public static final String MEMORY_POOL_RATIO    = "parquet.memory.pool.ratio";
-  public static final String MIN_MEMORY_ALLOCATION = "parquet.memory.min.chunk.size";
+  public static final String MIN_MEMORY_ALLOCATION= "parquet.memory.min.chunk.size";
+  public static final String ENABLE_BLOOM_FILTER  = "parquet.enable.bloom.filter";
+  public static final String EXPACTED_ENTRIES     = "parquet.expected.entries";
+  public static final String IS_FPP_PROVIDED      = "parquet.bloom.filter.fpp.provided";
+  public static final String FPP_VALUE            = "parquet.bloom.filter.value";
```

Comment:
if the value is set  then it is provided right? We should simplify settings
also the constant does not match the name.
spell out what FPP stands for